### ISSUE TITLE
Feature Request: Save output as HTML

### ISSUE THREAD

**Issue by ajmeese7** [mxrch/GitFive#12]
Whenever I'm saving the output logs of the `gitfive` command, all I can do is copy the text, so it loses all its color and isn't as visually appealing when investigating the results later.

If there was an option (i.e. `--html`) to save the colored output so it can be viewed identically later, that would be something I would be interested in having in this tool.

**Comment by mxrch** [comment]
Hey, I don't think it's a good idea, colors aren't made using HTML/CSS so the param --html is nonsense for me.
If someone wants to keep the colors, they can redirect the stdout in a file.

**Comment by ajmeese7** [comment]
Fair enough, I understand where you're coming from. Would you be open to a PR for this in the future or are you entirely not interested?

On your second point, I tried redirecting the stdout to a file and all the colors were lost, if you have a way to do so that preserves the colors I would be very interested in seeing it in action.

**Comment by mxrch** [comment]
Would it be a new feature like a full HTML output, with pretty style etc, or just the raw output with colors ? (choice will not affect the final answer, just curious 😄)

On which plateform are you ? Windows doesn't seems to keep colors, but Linux does

**Comment by ajmeese7** [comment]
I'm in Linux, currently running Zorin OS which is a Ubuntu-based distro. I haven't seen any of the colors captured when I pipe to a file, either with a redirect or with `tee`.

And I like having pretty reports whenever possible, producing an automatic HTML/PDF dossier to present when reporting to non-technical folks makes life incredibly easy on my side of things, even if I just take a screenshot of it. Raw colors would be plenty for now, just to make it easier to visually parse, but going the full route to report generation is always a good thing IMHO.
